### PR TITLE
Switching to auth2

### PR DESCRIPTION
**Pull request by rsutormin**

### CODE REVIEW

**Comment by rsutormin** [comment]
@msneddon: can you review this PR?

File: lib/DataFileUtil/DataFileUtilClient.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def __init__(
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
             auth_svc='https://kbase.us/services/authorization/Sessions/Login',
-            service_ver='release',
+            service_ver='dev',
```
Comment (msneddon) [inline lib/DataFileUtil/DataFileUtilClient.py]: We should keep this pointing to release.

File: lib/kb_ea_utils/kb_ea_utilsClient.py [hunk 2]
```diff
@@ -25,11 +25,11 @@ def __init__(
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
             auth_svc='https://kbase.us/services/authorization/Sessions/Login',
-            service_ver='release',
+            service_ver='dev',
```
Comment (msneddon) [inline lib/kb_ea_utils/kb_ea_utilsClient.py]: here too

**Comment by rsutormin** [comment]
@msneddon: Fixed

**Pull request merged**